### PR TITLE
Upgrade webmock to make feature spec compatible with Ruby 2.4

### PR DESCRIPTION
**Pull request by punitcse**

### CODE REVIEW

**Comment by punitcse** [comment]
I understand that the gem is set up to run with ruby 2.3.1  [doc](https://github.com/thoughtbot/administrate/blob/master/CONTRIBUTING.md) . But can we still consider this PR as most people want to run with the latest ruby version(2.4).

thanks :bowing_man: .

File: Gemfile.lock [hunk 1]
```diff
@@ -188,6 +189,7 @@ GEM
       slop (~> 3.4)
     pry-rails (0.3.4)
       pry (>= 0.9.10)
+    public_suffix (2.0.5)
```
Comment (BenMorganIO) [inline Gemfile.lock]: Not a new gem... :p

**Comment by BenMorganIO** [comment]
@punitcase see #793 for us bumping ruby version

File: Gemfile.lock (hunk 1 above)
Comment (punitcse) [inline Gemfile.lock]: @BenMorganIO `public_suffix` gem is the dependency for addressable gem. Please refer https://github.com/thoughtbot/administrate/pull/822/files#diff-e79a60dc6b85309ae70a6ea8261eaf95R55 . Do we still want to remove it?

Comment (BenMorganIO) [inline Gemfile.lock]: Nah, just some tongue and cheek humor from moi. I've been working a lot on making our Gemfile smaller such as removing some rails dependencies we don't use (such as actionmailer) to getting rid of some development dependencies (such as web-console).

**Pull request merged**